dumper.go

```diff
@@ -198,8 +198,16 @@ func (d *Dumper) dump(val reflect.Value, ignoreDepth ...bool) {
 		d.dumpMap(val)
 	case reflect.Func:
 		d.buf.WriteString(__(d.Theme.Func, val.Type().String()))
+		if val.IsNil() {
+			d.buf.WriteString(__(d.Theme.Braces, "(") + __(d.Theme.Nil, "nil") + __(d.Theme.Braces, ")"))
```

Comment:
Based on the fact you are repeating the sane pattern multiple times, can't it be refactored to define one functional and call it everywhere?

Reply:
I thought it would be good to keep it verbose, But, I'll refa tor that.

Reply:
I don't like to create colored functions that do one simple operation.

So, the best option I have on my head is to create a private property that holds that string and keep using it everywhere.